### DISCUSSION TITLE
Use --keep-tag along for retention policy?

### DISCUSSION THREAD

**Discussion by andreiz** [garethgeorge/backrest#533] [Q&A]
I can't seem to figure out how to add `--keep-tag <tag>` to the `forget` command that backrest runs as part of retention policy. Any clues?

**Comment by OldManBrodie** [comment]
I'm also curious about this. I just had to redo my entire system, and I want to make sure my last good backup is retained forever, or at least until I'm sure I don't need it anymore. I don't really want to change my whole retention policy for just one snapshot. I'd like to just manually tag it and then have backrest use the "--keep-tag" flag.
